app/views/layouts/application.html.erb

```diff
@@ -93,6 +93,12 @@
     </script>
     <!-- End Piwik Code -->
 
+    <script type="text/javascript">
+      if("<%= ENV["RAILS_ENV"] %>" !== "development" && typeof window.__REACT_DEVTOOLS_GLOBAL_HOOK__ === "object") {
+        window.__REACT_DEVTOOLS_GLOBAL_HOOK__.inject = function () {}
```

Comment:
This disables the React Dev Tools on any environment that is not "development" https://github.com/facebook/react-devtools/issues/191#issuecomment-454765533